pkg/component/plutono/plutono_test.go

```diff
@@ -792,6 +803,25 @@ func testDashboardConfigMap(ctx context.Context, c client.Client, namespaceName
 	ExpectWithOffset(1, len(availableDashboards)).To(Equal(dashboardCount))
 }
 
+func getDashboardConfigMaps(c client.Client, ctx context.Context) ([]corev1.ConfigMap, error) {
+
+	configMapList := &corev1.ConfigMapList{}
+	err := c.List(ctx, configMapList)
+	if err != nil {
+		return configMapList.Items, err
```

Comment:
```suggestion
	if err := c.List(ctx, configMapList); err != nil {
		return nil, err
```

Also, doesn't it make sense to restrict the list to a single namespace?